common/src/main/java/me/shedaniel/architectury/event/events/BlockEvent.java

```diff
@@ -32,32 +32,58 @@
 import org.jetbrains.annotations.Nullable;
 
 public interface BlockEvent {
-    
-    // Block interaction events
     /**
-     * Called when a player breaks a block.
+     * @see Break#breakBlock(Level, BlockPos, BlockState, ServerPlayer, IntValue)
```

Comment:
(For all @see for event fields)
Do we even need these, these seem pretty redundant

Reply:
When someone looks for usage they might find this event field instead of the called method and with this @see they can directly click to the corresponding method